lit/v5.2.0.lit

```diff
@@ -106,6 +110,28 @@
   the dashboard page by setting \code{cluster-name} flag.
 }
 
+\note{feature}{
+  \ghuser{cappyzawa} added a new `get-team` subcommand to `fly`. It allows you to
```

Comment:
```suggestion
  \ghuser{cappyzawa} added a new \reference{fly-get-team} command. It allows you to
```

(This will fail to build until we document `fly-get-team`, fyi.)